src/ai/backend/manager/services/vfolder/services/file.py

```diff
@@ -179,7 +185,9 @@ async def list_files(self, action: ListFilesAction) -> ListFilesActionResult:
         )
 
     async def rename_file(self, action: RenameFileAction) -> RenameFileActionResult:
-        allowed_vfolder_types = await self._shared_config.get_vfolder_types()
+        allowed_vfolder_types = (
+            await self._unified_config.legacy_etcd_config_loader.get_vfolder_types()
```

Comment:
I know we should replace this, but I leave it as it is for now to minimize side effects.